src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,474 @@
+		System.out.printf("| Username            | Fullname            | Department            | Address             | Role            |%n");
+		System.out.format("+---------------------+---------------------+-----------------------+---------------------+-----------------+%n");
+		for (Employee emp : empList) {
+			System.out.format(
+				leftAlignFormat, emp.getUsername(), emp.getFullname(),
+				emp.getDepartment(), emp.getAddress(), emp.getRole());
+		}
+		System.out.format("+---------------------+---------------------+-----------------------+-----------------------+----------------+%n");
+		System.out.println("\n");
+	}
+
+
+	/**
+	 * This method is used to delete employee using user input
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void deleteUser(Scanner scanner) {
+
+		System.out.println("Do you want to delete User? Please enter the fullname::\n");
+		String fullname = ValidationUtils.emptyValidation(scanner, ConstantUtils.FULLNAME);
+		employeeServiceImpl = new EmployeeServiceImpl();
+		boolean isDeleted = employeeServiceImpl.delete(fullname);
+		if (isDeleted) {
+			System.out.println(fullname + " is deleted successfully.\n");
+		}
+	}
+
+	/**
+	 * This method is used to terminate employee, delete use
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void terminateUser(Scanner scanner) {
+
+		System.out.println("Do you want to terminate User? Please enter the fullname::\n");
+		String fullname = ValidationUtils.emptyValidation(scanner, ConstantUtils.FULLNAME);
+		employeeServiceImpl = new EmployeeServiceImpl();
+		boolean isTerminated = employeeServiceImpl.terminate(fullname);
+		if (isTerminated) {
+			System.out.println(fullname + " is terminated successfully.\n");
+		}
+
+	}
+
+	/**
+	 * This method is used to display edit employee options
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void editOptions(Scanner scanner) {
+
+		System.out.println("*****************************************"
+			+ "\n" + "**   1. Full Name                             **" + "\n"
+			+ "**   2. Department                            **" + "\n"
+			+ "**   3. Address                               **" + "\n"
+			+ "******************************************" + "\n");
+	}
+
+	/**
+	 * This method is used to edit employee information
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @param id
+	 *            {@link int}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void editEmployee(Scanner scanner, int id) {
+
+		System.out.println("Do you want to edit your information? Please choose the options::\n");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)